_episodes/03-motion.md

```diff
@@ -1054,7 +1055,7 @@ pm_mask = (between(pm1, pm1_min, pm1_max) &
 ~~~
 {: .language-python}
 
-Again, the sum of a Boolean series is the number of `True` values.
+Again, the sum of a Boolean series is the number of `TRUE` values.
```

Comment:
```suggestion
Again, the sum of a Boolean series is the number of `True` values.
```

Reply:
Remark, if this true is in python, it would be "True". If it is a MySQL true, than it could be TRUE...